data/json/items/tool/metalworking.json

```
@@ -85,6 +85,44 @@
     "flags": [ "DURABLE_MELEE" ],
     "melee_damage": { "bash": 40 }
   },
+  {
+    "id": "anvil_scavenged",
+    "type": "ITEM",
+    "subtypes": [ "TOOL" ],
+    "name": { "str": "scavenger's anvil" },
+    "description": "A portion of heavy steel machinery that has been smoothed and shaped to form a very adequate anvil.",
+    "//": "Represents any number of heavy machinery parts that serve this purpose.",
+    "weight": "50 kg",
+    "volume": "30 L",
+    "longest_side": "40 cm",
+    "price": "1 kUSD",
+    "price_postapoc": "20 USD",
+    "to_hit": { "grip": "bad", "length": "hand", "surface": "any", "balance": "clumsy" },
+    "material": [ "steel" ],
```

```suggestion
    "material": [ "steel" ],
    "qualities": [ [ "ANVIL", 2 ] ],
```
I think you forgot the quality there